editor/plugins/sprite_frames_editor_plugin.cpp

```diff
@@ -1228,6 +1243,7 @@ void SpriteFramesEditor::_update_library(bool p_skip_selector) {
 		if (selected) {
 			animations->scroll_to_item(selected);
 		}
+		animations_dirty = false;
```

Comment:
This is checked only once, so it can be moved to the top right after `animations->clear();`, to avoid the other assign.